src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -175,17 +204,19 @@ def _read_tool_result(result_dir, batch_dir, kbase_collection, load_ver, tool_fi
     # retrieve and process the genome metadata file
     metadata_file = os.path.join(batch_result_dir, loader_common_names.GENOME_METADATA_FILE)
     try:
-        meta_df = pd.read_csv(metadata_file, sep='\t', names=['tool_identifier', 'genome_id', 'source_file_path'])
+        meta_df = pd.read_csv(metadata_file, sep='\t')
     except Exception as e:
         raise ValueError('Unable to retrieve the genome metadata file') from e
+    
     tool_genome_map = dict(zip(meta_df.tool_identifier, meta_df.genome_id))
+    tool_upa_map = _create_tool_upa_map(meta_df.tool_identifier, meta_df.source_dir, meta_df.meta_filename)
```

Comment:
it's better to do meta_df[_META_TOOL_IDENTIFIER], meta_df[_DATA_ID_COLUMN_HEADER] here. 
We should probably also move those variables to `loader_common_names`

Reply:
Moved to loader_common_names, and updated computation & parsing script.